.trunk/trunk.yaml

```diff
@@ -0,0 +1,22 @@
+version: 0.1
+cli:
+  version: 0.7.0-beta
+lint:
+  enabled:
+    - actionlint@1.6.8
+    - eslint@8.9.0
+    - gitleaks@8.3.0
+    - markdownlint@0.31.1
+    - prettier@2.5.1
```

Comment:
Should this stay in sync with `package.json`?

Reply:
Yeah, ideally they should stay in sync if you run any of these outside of trunk.